mysql-test/include/audit_log_events.inc

```diff
@@ -37,6 +37,9 @@ alter table t1 rename renamed_t1;
 select * from t_doesnt_exist;
```

Comment:
This is audit log format-independent test, maybe put pull this up from an include to some testcase, i.e. audit_log.test